shared/chat/conversation/messages/react-button/emoji-picker/index.tsx

```diff
@@ -30,9 +32,9 @@ const getData = memoize((topReacjis: Array<string>) => {
   // SectionList data is mostly static, map categories here
   // and chunk data within component
   const emojiSections = allCategories.map(c => ({
-    category: c.category,
```

Comment:
what are these non section list typing changes?

Reply:
These are changes to make the section list in this file better typed.